composer.json

```diff
@@ -58,7 +58,8 @@
         "symfony/process": "^3.4 || ^4.0",
         "symfony/yaml": "^3.4 || ^4.0",
         "thecodingmachine/safe": "^0.1.16",
-        "webmozart/assert": "^1.3"
+        "webmozart/assert": "^1.3",
+        "webmozart/path-util": "^2.3"
```

Comment:
I'm not super found of depending on it, but I think it's ok for now. Long term plan would be https://github.com/symfony/symfony/pull/30969 if it lands